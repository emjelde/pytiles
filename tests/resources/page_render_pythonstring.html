<html>
	<head>
		<title>%(title)s</title>
	</head>
	<body>
		Hello, %(name)s
	</body>
</html>
