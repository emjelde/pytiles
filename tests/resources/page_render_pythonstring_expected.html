<html>
	<head>
		<title>Welcome</title>
	</head>
	<body>
		Hello, pyTiles
	</body>
</html>
